Dockerfile

```diff
@@ -23,6 +23,7 @@ ENV PHP_SNAPPY_VERSION="bfefe4906e0abb1f6cc19005b35f9af5240d9025"
 ENV PHP_LZ4_VERSION="2f006c3e4f1fb3a60d2656fc164f9ba26b71e995"
 ENV PHP_XZ_VERSION=5.2.7
 ENV PHP_EXT_XZ_VERSION=1.1.2
+ENV PHP_ZOPFLI_VERSION="master"
```

Comment:
Let's use a specific commit hash instead of master.  LIke this how we do brotli.

Reply:
Used the commit hash for the latest commit in extension's repository @stnguyen90